src/views/Graph.vue

```diff
@@ -249,13 +276,51 @@ export default {
       graphID: null,
       // instance of system which provides pan/zoom/navigation support
       panZoomWidget: null,
-      // if true layout is left-right is false it is top-bottom
-      transpose: false,
-      // if true the graph layout will be updated on a timer
-      autoRefresh: true,
       // true if layout is in progress
       updating: false,
-      controlGroups: [
+    }
+  },
+
+  mounted () {
+    if (!this.autoRefresh) {
+      // load the graph if the autorefresh is off
+      this.refreshTimer = setInterval(this.refresh, 1000)
```

Comment:
For now how about this:

```suggestion
      this.refreshTimer = setInterval(this.refresh, 1000)
      this.refreshTimer = setInterval(this.refresh, 10000)  // make sure the graph is refreshed even if data is slow to arrive
```

Crude but effective.

Punt the rest to https://github.com/cylc/cylc-ui/issues/1756.

Reply:
Hmm, I thought this was fixed with this pr https://github.com/markgrahamdawson/cylc-ui/pull/3/files
That was merged into this branch? But the changes don't seem to be included?

Reply:
That commit doesn't appear to be on this branch any more.

I think you [force pushed](https://github.com/cylc/cylc-ui/compare/a4dcf253506e89eea93045f235657dbc495ead57..9abf4dce0da7fc6cce6fac4aa9e43134d704eb11#diff-327c8ac58a04b20db03a8d9e07c142db2a3627d32226f9bb1c0855e03dba0bc2R284) over it because it was added to the remote branch but not the local one.

Here's one way to pull upstream changes (e.g. code suggestions, side-prs, etc) into your local branch

```console
$ git checkout <branch>
$ git fetch origin
$ git rebase origin/<branch>
```

Before force-pushing I often check the diff against origin like this:

```console
$ git fetch origin
$ git diff origin/<branch>
$ git push -f origin/<branch>
```